test/extended/util/cli.go

```diff
@@ -36,10 +40,23 @@ type CLI struct {
 	finalArgs       []string
 	stdout          io.Writer
 	verbose         bool
+	execBashArgs    bool
 	cmd             *cobra.Command
 	kubeFramework   *e2e.Framework
 }
 
+// ExitError happens when a command running inside of a container launched
+// using oc exec exits with non-zero code.
+// TODO: Remove this when https://github.com/kubernetes/kubernetes/pull/13728 is merged in OS
+type ExitError struct {
```

Comment:
https://github.com/openshift/origin/pull/4771

when this gets merged, you will not need this.